dotCMS/src/main/java/com/dotcms/rest/api/v1/browsertree/BrowserTreeResource.java

```diff
@@ -62,6 +62,48 @@ public BrowserTreeResource(final WebResource webResource,
     }
 
     @GET
+    @Path ("/sitename/{sitename}/uri/")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public final Response loadAssetsUnder(
+            @Context final HttpServletRequest req,
+            @PathParam("sitename")   final String sitename
+    ) {
+        Response response = null;
+        final InitDataObject initData = this.webResource.init(null, true, req, true, null);
+        final User user = initData.getUser();
+        final List<Map<String, Object>> assetResults;
+
+        try {
+            Locale locale = LocaleUtil.getLocale(user, req);
+
+            assetResults = browserTreeHelper.getTreeablesUnder(sitename,user,"/")
+                    .stream()
+                    .map(treeable -> {
+                        try {
+                            return treeable.getMap();
+                        } catch (Exception e) {
+                            Logger.error(this,"Data Exception while converting to map", e);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Data Exception while converting to map" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)